nextjs-app/app/offline/page.tsx

```diff
@@ -50,13 +50,8 @@ export default async function OfflinePage(props: Props) {
 
   return (
     <>
-      <div className="container my-12 lg:my-24 grid gap-12">
-        <div className="pb-6 mb-6 border-b border-gray-100">
-          <h2 className="text-4xl font-bold tracking-tight text-gray-900 sm:text-5xl lg:text-7xl">
-            {offline.title}
-          </h2>
-
-        </div>
+      <div className="h-[200px] md:h-[250px]" />
```

Comment:
if you need top margin, it can be added to the following element rather than adding an empty spacer div